code/modules/surgery/organic_steps.dm

```diff
@@ -138,3 +138,37 @@
 		"<span class='notice'>[user] drills into [target]'s [parse_zone(target_zone)]!</span>",
 		"<span class='notice'>[user] drills into [target]'s [parse_zone(target_zone)]!</span>")
 	return ..()
+
+//pick bone
+/datum/surgery_step/pick
```

Comment:
why are these two steps outside of the transorbital lobotomy file? they're not used in any other surgeries.